var/spack/repos/spack_repo/builtin/packages/py_melissa_core/package.py

```diff
@@ -16,47 +17,86 @@ class PyMelissaCore(PythonPackage):
 
     homepage = "https://gitlab.inria.fr/melissa/melissa"
     git = "https://gitlab.inria.fr/melissa/melissa.git"
-    maintainers("robcaulk", "mschouler", "raffino")
+    # FIXME: Replace with an official link
+    url = "https://gitlab.inria.fr/melissa/melissa/-/archive/ac-develop-stable/melissa-ac-develop-stable.tar.gz"
+    maintainers("abhishekp1297", "viperML", "raffino")
 
     license("BSD-3-Clause")
 
     version("develop", branch="develop", preferred=True)
-    version("joss", tag="JOSS_v2", commit="20bbe68c1a7b73aa2ea3ad35681c332c7a5fc516")
-    version("sc23", tag="SC23", commit="8bb5b6817d4abe4eaa5893552d711150e53535f3")
+    version(
+        "joss", tag="JOSS_v2", commit="20bbe68c1a7b73aa2ea3ad35681c332c7a5fc516", deprecated=True
+    )
+    version("sc23", tag="SC23", commit="8bb5b6817d4abe4eaa5893552d711150e53535f3", deprecated=True)
 
     # define variants for the deep learning server (torch, tf)
     variant(
         "torch", default=False, description="Install Deep Learning requirements with Pytorch only"
     )
     variant(
-        "tf", default=False, description="Install Deep Learning requirements with TensorFlow only"
+        "tf",
+        default=True,
+        when="~torch",
+        description="Install Deep Learning requirements with TensorFlow only",
     )
-
-    depends_on("c", type="build")  # generated
```

Comment:
Removing this and the next line will result in not having a c or fortran compiler for this package. Is it only using python? The fact that the lines were added seems to indicate it has c and fortran code.